openmls/src/group/mls_group/errors.rs

```diff
@@ -78,10 +66,14 @@ pub enum EmptyInputError {
 /// Group state error
 #[derive(Error, Debug, PartialEq, Clone)]
 pub enum MlsGroupStateError {
+    #[error(transparent)]
+    LibraryError(#[from] LibraryError),
     #[error("Tried to use a group after being evicted from it.")]
     UseAfterEviction,
     #[error("Can't create message because a pending proposal exists.")]
     PendingProposal,
+    #[error("Can't execute operation because a pending commit exists.")]
+    PendingCommit,
```

Comment:
Is it always clear which operation didn't succeed or do we need to put the operation in here?

Reply:
A number of operations can fail when there is a pending commit, like adding & removing members. This is an immediate result of the call, so it should be clear from the context.